src/main/java/com/aquent/viewtools/PostTool.java

```diff
@@ -229,9 +229,9 @@ public PostToolResponse sendStringData(String url, String data) {
    * @param creds - A credentials object for authenticated requests.
    * @return A PostToolResponse Object
    */
-  public PostToolResponse send(String url, Map<String, String> params, String method, String contentType, Credentials creds) {
-    Logger.debug(this, "send(Map) called with url=" + url + ", params=" + params + ", and method=" + method
-        + ", contentType = " + contentType + ", creds=" + UtilMethods.isSet(creds));
+  public PostToolResponse send(String url, Map<String, String method, String contentType) {
```

Comment:
Not quite right here .. this is where having an IDE would help.  Again, not removing the parameters from the method, just the logging statements, but this line straight up won't compile.